### PR TITLE
added controlled vocabulary yaml

### PR DESCRIPTION
**Pull request by sbello**
Added file with the controlled vocabulary LinkML model
@chris-grove would you please review and let me know if it needs any corrections

### CODE REVIEW

**Review by chris-grove — APPROVED** [review]
Thanks @sbello !
I've cleaned up a few things here and there, ran 'make all' locally and everything seems to be in order.

**Comment by chris-grove** [comment]
@sierra-moxon 
Could you just take a look at the header for this new 'controlledVocabulary.yaml' file and see that is looks OK?

File: model/schema/ontologyTerm.yaml [hunk 1]
```diff
@@ -56,6 +54,12 @@ classes:
     defining_slots:
       - curie
       - namespace
+    slot_usage:
+      - definition:
```
Comment (sierra-moxon) [inline model/schema/ontologyTerm.yaml:58]: - definition doesn't need the dash.

**Comment by sierra-moxon** [comment]
looks great - just make sure to add the new yaml file to allianceModel.yaml -- as that is the "root" of our project that is used by all the generators to traverse our yaml files. 

https://github.com/alliance-genome/agr_curation_schema/blob/main/model/schema/allianceModel.yaml

**Comment by chris-grove** [comment]
@sierra-moxon 
OK, thanks for catching those! Running 'make all' now to check and will merge after that.

**Pull request merged**